src/main/java/thomasmccue/dbclientapp/dao/CountryDao.java

```diff
@@ -0,0 +1,27 @@
+package thomasmccue.dbclientapp.dao;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import thomasmccue.dbclientapp.helper.JDBC;
+
+import java.sql.*;
+public class CountryDao {
+        public static ObservableList<String> getAllCountries() {
+            String sql = "SELECT Country FROM client_schedule.countries";
+            ObservableList<String> results = FXCollections.observableArrayList();
+
+            try (Connection connection = JDBC.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+                try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                    while (resultSet.next()) {
+                        results.add(resultSet.getString(1));
+                    }
+                }
```

Comment:
Curious about this additional try inside of the outer try and what the pattern is here? I see you seem to be using the initial try for the connection scope, do the ResultSet need to be closed too?